drivers/led/sn32/rgb_matrix_sn32f24xb.c

```diff
@@ -273,10 +271,13 @@ static void update_pwm_channels(PWMDriver *pwmp) {
             matrix_read_rows_on_col(shared_matrix, current_key_col, row_shifter);
         }
     }
-#ifdef LED_COLUMN_INVERSE_LEVEL
+#ifdef MATRIX_UNSELECT_DRIVE_HIGH
+#if (RGB_OUTPUT_ACTIVE_LEVEL == PWM_OUTPUT_ACTIVE_HIGH)
```

Comment:
this can probably become `#if ((RGB_OUTPUT_ACTIVE_LEVEL == PWM_OUTPUT_ACTIVE_HIGH ) && (RGB_OUTPUT_ACTIVE_LEVEL == PWM_OUTPUT_ACTIVE_HIGH))

Reply:
Turned this into
`#if ((RGB_OUTPUT_ACTIVE_LEVEL == RGB_OUTPUT_ACTIVE_HIGH) && defined(MATRIX_UNSELECT_DRIVE_HIGH))`
For this logic it looks right, we need to turn all columns low only when matrix unselect high and RGB output high too, so they don't conflict with each other.